tuneme/templates/comments/report_response.html

```diff
@@ -0,0 +1,20 @@
+{% extends "core/_article_page.html" %}
+{% load wagtailcore_tags wagtailimages_tags comments mptt_tags molo_commenting_tags i18n %}
+
+{% block content %}
+
+<div class="title {{article.get_parent_section.get_effective_extra_style_hints}}">
+
+</div>
+
+  <h4 class="left">Thanks!</h4>
+  
+<div class="block {{article.get_parent_section.get_effective_extra_style_hints}}">
+  <div class="article-copy">
+    <p><strong>This comment has been reported.</strong>
+    It will be reviewed by the Tune Me team 
+    and may be removed if it breaks our Platform rules</p>
+  </div>
```

Comment:
The above text should really be in the locale file or wagtail or something. @miltontony please advise